### DISCUSSION TITLE
Icon request: fa fa-molecule

### DISCUSSION THREAD

**Discussion by kalatchev** [FortAwesome/Font-Awesome#21140] [Icon Requests]
It's will be great to have an icon for chemistry related stuff, like molecule. Something [like that](http://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Benzoic-acid.svg/100px-Benzoic-acid.svg.png).

**Comment by respectyoda** [comment]
+1 I would prefer for it to look like this: <a href="http://www.shutterstock.com/pic.mhtml?id=227588569&src=lb-31441375">cool molecule</a>

**Comment by kalatchev** [comment]
Yes, both are good. I think both concepts should be implemented - planetar model as well as chemical formula. Another good option for such cases is atom with electrons, [like this one](http://icongal.com/gallery/image/5890/nuclear_atom_atomic.png). It is quite popular and recognisable symbol.

**Comment by RileyStarlight** [comment]
+1
Atom w/ electrons is worldwide recognisable.

**Comment by nathanmarks** [comment]
+1 really would find this useful for academic applications.

However I really would like a simple ball and stick molecule, not an atom with electrons. (a la http://www.middleschoolchemistry.com/img/content/lessons/2.2/ball_and_stick_water_molecule.jpg).

**Comment by nathanmarks** [comment]
@kalatchev @kwendenarmo An atom should be fa-atom not fa-molecule.

**Comment by RileyStarlight** [comment]
For sure!

**Comment by nathanmarks** [comment]
Although I did just realise, how different would a stick and ball icon be from `fa-share-alt`? Enough to warrant another icon? I'm not so sure. I'm currently using the `fa-share-alt` icon for a molecule.

**Comment by nathraQ** [comment]
+1

**Comment by A1fus** [comment]
+1

**Comment by G-Driver** [comment]
+1

**Comment by ktreads** [comment]
+1

**Comment by janani-s** [comment]
+1

**Comment by xvelasqu** [comment]
+1

**Comment by evanschris** [comment]
+1

**Comment by janvanderhaegen** [comment]
+1, 
Not needing [this](http://www.shutterstock.com/pic.mhtml?id=227588569&src=lb-31441375) to represent a molecule, but for different 'connected modules'. It serves a double purpose, really :)

**Comment by robbyclark** [comment]
OK, I have designed an SVG for this already (although I had to upload PNG due to file restrictions) - I needed it for a project myself and had to create a custom font set, but would be delighted if this was added to font awesome!

![molecular](https://cloud.githubusercontent.com/assets/14829991/10090565/d9a89b1a-6328-11e5-9528-91fa6189202c.png)

**Comment by AndreLZGava** [comment]
+1

**Comment by stas-s** [comment]
+1

**Comment by egost** [comment]
+1

**Comment by mingushead** [comment]
+1

**Comment by dridk** [comment]
+1

**Comment by optimum-web** [comment]
+1

**Comment by DERNERSERFT** [comment]
+1

**Comment by Maximvdw** [comment]
+1

**Comment by sergiobellon** [comment]
+1

**Comment by pierreavn** [comment]
+1

**Comment by radusuciu** [comment]
Would also like to see something like this:

![image](https://user-images.githubusercontent.com/1108600/88835398-0d79ce80-d18a-11ea-918a-92e287a0b0b9.png)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
